modules/oe_whitelabel_search/src/Form/SearchForm.php

```diff
@@ -48,55 +48,30 @@ public function buildForm(array $form, FormStateInterface $form_state, array $co
     $form_state->set('oe_whitelabel_search_config', $config);
     $input_value = '';
 
+    $form['#region'] = $config['form']['region'];
     if (!empty($config['input']['name'])) {
```

Comment:
looking at it, the field is required, so wont be empty the way we use it, still the form could be injected so I think its a better idea to return here if empty, since the name should not be empty ever